static/app/components/button.tsx

```diff
@@ -18,7 +18,7 @@ type ButtonElement = HTMLButtonElement & HTMLAnchorElement & any;
 
 type Props = {
   priority?: 'default' | 'primary' | 'danger' | 'link' | 'success' | 'form';
-  size?: 'zero' | 'xsmall' | 'small';
+  size?: 'zero' | 'xsmall' | 'small' | 'default';
```

Comment:
I'm not sure if we should provide the `default` type here, because if the user doesn't provide that prop, it will fall back to `default`, right? if we introduce the `default` type here, maybe some developer uses it explicitly

Reply:
ahh added this just to be complete, but you have a point. Just removed it!

Reply:
thanks!